nvflare/tool/poc/poc_commands.py

```diff
@@ -680,7 +708,9 @@ def _get_clients(service_commands: list, service_config) -> List[str]:
     return clients
 
 
-def _build_commands(cmd_type: str, poc_workspace: str, service_config, excluded: list, services_list=None) -> list:
+def _build_commands(
+    cmd_type: str, poc_workspace: str, service_config, project_config, excluded: list, services_list=None
+) -> list:
     """Builds commands.
 
```

Comment:
Is there a better name for this method?
get_all_commands? or get_service_commands?

Another question why does this method start with underscore
But sort_service_cmds does not?